src/encoded/schemas/gene.json

```diff
@@ -0,0 +1,165 @@
+          "type": "string",
+          "default": "live",
+          "enum" : [
+              "live",
+              "secondary",
+              "discontinued"
+          ],
+          "permission": "import_items"
+        },
+        "symbol": {
+            "title": "Gene symbol",
+            "description": "Gene symbol approved by the official nomenclature.",
+            "type": "string",
+            "format": "gene_name",
+            "permission": "import_items"
+        },
+        "name": {
+            "title": "Gene name",
+            "description": "The full gene name preferably approved by the official nomenclature.",
+            "type": "string",
+            "format": "gene_name",
+            "permission": "import_items"
+        },
+        "synonyms": {
+            "title": "Synonyms",
+            "description": "Alternative symbols that have been used to refer to the gene.",
+            "comment": "The list of synonyms is maintained using information from NCBI Entrez. Synonyms not included in NCBI gene_info can be added but are not actively maintained.",
+            "type": "array",
+            "uniqueItems": true,
+            "items": {
+                "title": "Synonym",
+                "description": "An alternative symbol/name referring to the gene.",
+                "type":  "string",
+                "format": "gene_name"
+            },
+            "permission": "import_items"
+        },
+        "dbxrefs": {
+            "title": "External identifiers",
+            "description": "Unique identifiers from external resources.",
+            "comment": "This list should have at least an ID from the official nomenclature. The list of other dbxrefs is maintained using information from NCBI Entrez. Identifiers not included in NCBI gene_info can be added but are not actively maintained.",
+	          "type": "array",
+            "default": [],
+            "uniqueItems": true,
+            "items": {
+                "title": "External identifier",
+                "description": "A unique identifier from external resource (e.g. HGNC, MGI, FlyBase, WormBase, ENSEMBL, MIM, UniProtKB, Vega, miRBase).",
+                "comment": "Submit as database_name:id. See changelog for potential databases.",
+                "type":  "string",
+                "pattern": "^((HGNC:HGNC:\\d+)|(MGI:MGI:\\d+)|(FlyBase:FBgn\\d+)|(WormBase:WBGene\\d+)|(ENSEMBL:ENS[A-Z]*G\\d+)|(MIM:\\d+)|(UniProtKB:[0-9A-Z]+)|(Vega:OTT[A-Z]+G\\d+)|(miRBase:MI\\d+)|(IMGT/GENE-DB:[0-9A-Za-z/\\(\\)\\-]+))$"
+            },
+            "permission": "import_items"
+        },
+        "go_annotations": {
+            "title": "Gene Ontology annotations",
+            "descrption": "Gene ontology annotations of the gene, including all three aspects: Biological Process, Cellular Component, and Molecular Function.",
+            "comment": "Data imported from the Gene Ontology Consortium.",
+            "type": "array",
```

Comment:
Maybe we also need a default empty [] value for this property as it doesn't seem to show up in objects that don't have any GO terms in this property (even in the inserts that have an empty list value). That way if people are operating on these objects to pull out GO annotations, they would not get key errors.